test/Splunk.OpenTelemetry.AutoInstrumentation.Tests/LoggerTests.cs

```diff
@@ -0,0 +1,37 @@
+// <copyright file="LoggerTests.cs" company="Splunk Inc.">
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using Splunk.OpenTelemetry.AutoInstrumentation.Logging;
+
+namespace Splunk.OpenTelemetry.AutoInstrumentation.Tests;
+
+public class LoggerTests
+{
+    [Fact]
+    public void ConstructorDoesNotThrowExceptionWhenReflectionFails()
+    {
+        var exception = Record.Exception(() => new Logger());
+        exception.Should().Be(null);
+    }
```

Comment:
I think that this test is useless

1. We have a default constructor
2. `ImplementationDoesNotThrowExceptionWhenReflectionFails` would fail if `new Logger();` throws an exception

Reply:
At least it guarantees that updating Otel Auto version can not break the whole solution with a logger API change :)

Reply:
I mean that `ImplementationDoesNotThrowExceptionWhenReflectionFails` guarantees  the same 😉